06-html5-apis/00_Storage_DnD_WebSockets/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>HTML APIs</title>	
+	<link rel="stylesheet" type="text/css" href="css/index.css">
+	<script type="text/javascript" src="js/index.js"></script>
+</head>
+
+<body>
+	<div class="container">
+		<section>
+		<fieldset>
+			<h1> Insert a new Player here! </h1>
+			<label for="name"> Player Name:</label><br>
+			<input id="playerName" type="text" name="name" placeholder="Insert player name"></input>
+			<br><br>
+			<label for="name"> Player Description:</label><br>
+			<textarea id="playerDescription" placeholder="Inser player description here"></textarea>
+			<br><br>
+			<button onclick="saveDataLocalStorage()">Save LocalStorage</button>
+			<button onclick="saveDataIndexDB()">Save DataIndexDB</button>			
+			<button onclick="clearSaveContent()"> Clear Saved Content</button>			
+		</fieldset>
+		<br><br>
+		</section>
+
+		<section>
+			<fieldset>
+			<h1>Drop text file here to add a new player!</h1>
+			<div id="drop_zone">
+				<output id="list"></output>
```

Comment:
Good use of html 5 tags... but seems not being supported by IE (http://www.w3schools.com/tags/tag_output.asp).